### ISSUE TITLE
Open Date Picker on Input Click.

### ISSUE THREAD

**Issue by Sal-man** [jkuri/ng2-datepicker#235]
Hi jkuri, is there any way to Open Date picker on Input Click and also on Calendar icon Click?

**Comment by raviji** [comment]
Yes i need the same for clicking input calendar should open in my case. Anyone help me on this?

**Comment by Sal-man** [comment]
@raviji  i achieve this through css, i have by giving the icons class padding-left and margin-left in negative so the design is not changed and you actually not clicked on input instead in this method you click on calender. here is the css.

`.datepicker-input-icon {
    margin-left: -400px;
    padding-left: 400px;
}`

**Comment by raviji** [comment]
I found a solution myself for this issue. 
`
openDate(){
    $('.datepicker-input-icon').click();
  }
`
and in the html call that function
`
<ng2-datepicker [options]="options" [(ngModel)]="date" (click)="openDate()"></ng2-datepicker>
`

**Comment by yogeshfcc** [comment]
I have tried the above solution. This does not works for me.

**Comment by jeevanhr7** [comment]
Hi is there any Way to implement this. I tried  above does't works. i trying to open the DatePicker onClick of Button.
